### DISCUSSION TITLE
What is the erlang ACL rule for ALL topics - for the MQTT broker "root" account?

### DISCUSSION THREAD

**Discussion by jayjupdhig** [emqx/emqx#6019] [Q&A]
Hi there

Version i use: Most current one 4.3.8

...i was successful to copy to existing ACL "dashboard" user - i replaced the "dashbboard" string with "root" and then i saw the ´$SYS´ topic in MQTT explorer. (-> https://github.com/thomasnordquist/MQTT-Explorer/ )

**%%-- User-defined {
{allow, {user, "root"}, subscribe, ["$SYS/#"]}.
{allow, {ipaddr, "127.0.0.1"}, pubsub, ["$SYS/#", "#"]}.
{deny, all, subscribe, ["$SYS/#", {eq, "#"}]}.
{allow, all}.
%%-- } User-defined**

BUT: What's the rule to access ALL topics (just for the MQTT server root account - sure!! ;-))

´[" ... , ... "]´ instead if $SYS-only ´["$SYS/#"]´ ?

Thank you very much for your feedbacks.

**Comment by zmstone** [comment]
converting this to a discussion.

**Comment by zmstone** [comment]
`{allow, {user, "root"}, subscribe, ["#"]}.`

**Comment by jayjupdhig** [comment]
Now it works, thank you very very much:

**%% -- J.M., 2021-10-28 {
{allow, {user, "root"}, subscribe, ["#"]}.
%% -- } J.M., 2021-10-28

{allow, {user, "dashboard"}, subscribe, ["$SYS/#"]}.
{allow, {ipaddr, "127.0.0.1"}, pubsub, ["$SYS/#", "#"]}.
{deny, all, subscribe, ["$SYS/#", {eq, "#"}]}.
{allow, all}.**

**Comment by jayjupdhig** [comment]
**TEST**

**Comment by jayjupdhig** [comment]
Bold font makes stress again...
